src/pages/EditMonitor.vue

```diff
@@ -144,8 +147,26 @@
                                 </div>
                             </div>
 
-                            <!-- Json Query -->
-                            <div v-if="monitor.type === 'json-query'" class="my-3">
+                            <!-- NUT -->
+                            <!-- For NUT Type -->
+                            <template v-if="monitor.type === 'nut'">
+                                <div class="my-3">
+                                    <label for="upsName" class="form-label">UPS {{ $t("Name") }}</label>
```

Comment:
I am unshure if a longer name like `name of the uninterruptible power supply` might be clearer and more importantly translate better.
Thoughts?